src/MachO/BinaryParser.tcc

```diff
@@ -297,7 +297,16 @@ void BinaryParser::parse_load_commands(void) {
           load_command = new LoadCommand{command};
           break;
         }
-
+      case LOAD_COMMAND_TYPES::LC_ENCRYPTION_INFO_64:
```

Comment:
fair enough. that being said MachOKit and similar projects are simply ignoring this value